components/log-viewer-webui/client/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "log-viewer-webui-client",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/index.jsx",
+  "scripts": {
+    "lint:check": "npx eslint --no-eslintrc --config package.json src webpack.config.js",
+    "lint:fix": "npx eslint --fix --no-eslintrc --config package.json src webpack.config.js",
+    "build": "webpack --define-process-env-node-env production",
+    "watch": "webpack serve"
```

Comment:
For consistency, can we rename this config as `start`? Let's continue the discussion in  `server/package.json`.